Changes

```diff
@@ -1,3 +1,11 @@
+10.4.x.x (relative to 10.4.5.0)
+========
+
+Fixes
+-----
+- ImageReader : Check for oiio version before calling read_native_deep_x
```

Comment:
It would probably be more useful if this said something like "Fixed compilation with OIIO 2.3".

Reply:
Updated to your suggestion but used "< 2.4" instead of 2.3 specifically (1.5.15 is really the version that we are interested in)